src/js/controller/instream-adapter.js

```diff
@@ -201,7 +201,7 @@ var InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
         if (skipoffset) {
             // Start background loading once the skip button is clickable
             _this.setupSkipButton(skipoffset, _options);
-            _backgroundLoadPosition = skipoffset;
+            _backgroundLoadPosition = typeof skipoffset === 'string' ? utils.seconds(skipoffset) : skipoffset;
         } else {
             // If no skipoffset is set, default to background loading 5 seconds before the end
             _backgroundLoadPosition = item.duration - 5;
```

Comment:
Ad duration may not be known until ad playback starts. This logic should really go into a time handler.